Core/Object Arts/Dolphin/Base/DelayTest.cls

```diff
@@ -0,0 +1,128 @@
+﻿"Filed out from Dolphin Smalltalk 7"!
+
+DolphinTest subclass: #DelayTest
+	instanceVariableNames: 'delay startTime'
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+DelayTest guid: (GUID fromString: '{0b7c6875-1051-4b80-b137-cd1c02754bcd}')!
+DelayTest comment: ''!
+!DelayTest categoriesForClass!Unclassified! !
+!DelayTest methodsFor!
+
+expectDelay: expectedMilliseconds for: nowMicroseconds
```

Comment:
Slightly neater would be to pass a Duration for the expected delay.

Reply:
Fair enough—I did it the way I did to avoid repeating "milliseconds" everywhere, but either tradeoff is fine.